host/4/bookworm/python/python312/python312-appservice.Dockerfile

```diff
@@ -86,7 +91,6 @@ ENV LANG=C.UTF-8 \
     DOTNET_USE_POLLING_FILE_WATCHER=true \
     HOST_VERSION=${HOST_VERSION} \
     ASPNETCORE_CONTENTROOT=/azure-functions-host \
-    LD_LIBRARY_PATH=/opt/python/3.12/lib:$LD_LIBRARY_PATH \
```

Comment:
Is this not necessary? I can't remember why it was added

Reply:
We don't need it. Tested it without this variable.
We do need the libpython3.12.so.1.0 which I have linked.

Reply:
Could you explain the thought process behind this change?  If it works it works but IMO this is worse than using the builtin LD LIB PATH.  Using a symlink to the SO is probably fine but if there is anything else in that path that is used in the dir we might need it could be hard to tell until we hit a runtime error that its necessary

![image](https://github.com/user-attachments/assets/3ba8b891-0408-4e1d-9109-f0f6bbb0c18d)


For instance in this image we see libpython3.so  its tiny and probably used nowhere. But if its part of some vague path thats only called in some situations we would encounter this issue only then. Just raising this as a concern. If you are confident I am fine with keeping it

Reply:
There are 3 reasons that I made this change:
1) While building this image with the open telemetry package, I got an error about a missing libpython3.12.so dependency. That suggest we need this to install packages to the image.
2) The python 3.12 consumption image has the same symlink. Thats what I got it from. Better to have it in both images for consistency.
3) The LD LIB PATH was not getting used. After building the image, there was a warning message for this line, saying its not used. Since it doesn't have any affect, might as well remove it from the image.

Reply:
We should only drive consistency across the images with the right approach. With that being said I think this is fine approving

Reply:
Thanks. Just wanted to point out that this is the right approach. Adding the LD LIB PATH was a no-op. Linking the  libpython was needed since now all the python libraries are in opt which is not the pattern we follow in functions. 
This change will be consistent with what we have in our consumption images,